tests/_plugins/ui/_impl/test_input.py

```diff
@@ -114,6 +137,13 @@ def test_range_slider_init() -> None:
         # cast to floats
         assert isinstance(num, float)
 
+    slider = ui.range_slider(steps=[1, 3, 6, 10, 17, 20], value=[3, 17])
```

Comment:
can we add a test for empty `steps` (not sure how we want to handle) and invalid bounds (value is not in steps)